pkg/analysis/output.go

```diff
@@ -56,6 +56,10 @@ func (a *Analysis) jsonOutput() ([]byte, error) {
 
 func (a *Analysis) textOutput() ([]byte, error) {
 	var output strings.Builder
+
+	// Print the AI provider used for this analysis
+	output.WriteString(fmt.Sprintf("AI Provider: %s\n", color.YellowString(a.AnalysisAIProvider)))
```

Comment:
Shouldn't we add this information in json output ?

Reply:
Yeah I can do that!